src/hardware/mixer.cpp

```diff
@@ -994,6 +1002,56 @@ AudioFrame MixerChannel::ApplyCrossfeed(const AudioFrame &frame) const
 	return {a.left + b.left, a.right + b.right};
 }
 
+MixerChannel::Sleeper::Sleeper(MixerChannel &c) : channel(c) {}
+
+void MixerChannel::Sleeper::Listen(const AudioFrame &frame)
+{
+	// Truncate fractions between (-1 and 1) to count them as zero
+	accumulated_noise += abs(static_cast<int16_t>(frame.left)) +
```

Comment:
Maybe mention that this works because the float sample data is normalised to the signed 16-bit integer range.

Reply:
Great point; adding!